### PR TITLE
Add ZOOM to ISIS SANS GUI

### PR DESCRIPTION
**Pull request by AntonPiccardoSelg**
For the commissioning which is happening at the moment we need to add ZOOM to the new ISIS SANS GUI.
This PR has the following changes.

1. It adds the ZOOM option to the instruments in the ISIS SANS GUI
2. It adds ZOOM to the ISIS instruments on the First Time Setup UI

**To test:**

1. Click `Help > First Time Setup`
2. Select `ISIS` for the `Default Facility`
3. Ensure that `ZOOM` can be selected from the `Default Instrument` selection
4. Open the ISIS SANS v2 experimental GUI. Ensure that ZOOM can be selected from the instruments.

No release notes required and no issue number

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/SANS/sans/gui_logic/gui_common.py [hunk 1]
```diff
@@ -81,7 +87,7 @@ def get_reduction_mode_from_gui_selection(gui_selection):
         return ISISReductionMode.Merged
     elif gui_selection == ALL:
         return ISISReductionMode.All
-    elif gui_selection == SANS2D_LAB or gui_selection == LOQ_LAB or gui_selection == LARMOR_LAB or gui_selection == DEFAULT_LAB:  # noqa
+    elif gui_selection == SANS2D_LAB or gui_selection == LOQ_LAB or gui_selection == LARMOR_LAB or gui_selection == ZOOM_LAB or gui_selection == DEFAULT_LAB:  # noqa
```
Comment (AnthonyLim23) [inline scripts/SANS/sans/gui_logic/gui_common.py]: should the comment be at the end of this line?

**Review by AnthonyLim23 — COMMENTED** [review]
code looks good. Need to test it

**Review by AnthonyLim23 — APPROVED** [review]
Works as described

File: scripts/SANS/sans/gui_logic/gui_common.py (hunk 1 above)
Comment (AntonPiccardoSelg) [reply scripts/SANS/sans/gui_logic/gui_common.py]: Thanks for testing :+1:.  The `#noqa` should be on the same line as the thing that causes the flake issue you want to disable, see https://stackoverflow.com/questions/5033727/how-do-i-get-pyflakes-to-ignore-a-statement?answertab=active#tab-top In this case I want everything in one line for increased readability.

Comment (AnthonyLim23) [reply scripts/SANS/sans/gui_logic/gui_common.py]: ok its good to go then

Comment (AntonPiccardoSelg) [reply scripts/SANS/sans/gui_logic/gui_common.py]: Thanks :+1:

**Pull request merged**